phones/tests/models_tests.py

```diff
@@ -315,37 +345,191 @@ def test_create_relaynumber_already_registered_with_service(
         code=21710,
     )
 
-    real_phone = "+12223334444"
-    RealPhone.objects.create(user=phone_user, verified=True, number=real_phone)
+    # Does not raise exception
+    relay_number = "+19998887777"
+    RelayNumber.objects.create(user=phone_user, number=relay_number)
+
+    mock_number_create.assert_called_once_with(
+        phone_number=relay_number,
+        sms_application_sid=settings.TWILIO_SMS_APPLICATION_SID,
+        voice_application_sid=settings.TWILIO_SMS_APPLICATION_SID,
+    )
+    mock_services.assert_called_once_with(settings.TWILIO_MESSAGING_SERVICE_SID[0])
+    mock_messaging_number_create.assert_called_once_with(
+        phone_number_sid=twilio_incoming_number_sid
+    )
     mock_messages_create.assert_called_once()
-    mock_messages_create.reset_mock()
+    assert caplog.messages == ["twilio_messaging_service"]
+    assert caplog.records[0].code == 21710
 
-    relay_number = "+19998887777"
-    relay_number_obj = RelayNumber.objects.create(user=phone_user, number=relay_number)
 
-    mock_number_create.assert_called_once()
-    call_kwargs = mock_number_create.call_args.kwargs
-    assert call_kwargs["phone_number"] == relay_number
-    assert call_kwargs["sms_application_sid"] == settings.TWILIO_SMS_APPLICATION_SID
-    assert call_kwargs["voice_application_sid"] == settings.TWILIO_SMS_APPLICATION_SID
+def test_create_relaynumber_full_service(
```

Comment:
quibble (non-blocking): change this to `test_create_relaynumber_all_services_full`?

When I first read this, I was expecting this to be the "fallback" test.